app/src/main/java/com/depromeet/team6/data/dataremote/datasource/TransitsRemoteDataSource.kt

```
@@ -30,6 +38,69 @@ class TransitsRemoteDataSource @Inject constructor(
+                                }
+
+                                else -> {
+                                    Timber.e("Unknown streaming data format: $jsonString")
+                                    throw ApiException.NetworkFailureException.UnknownFailure("Unknown streaming data format: $jsonString")
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)